### PR TITLE
03 -- The Todo Towers

### PR DESCRIPTION
**Pull request by al-the-x**
## Rituals (~1h 15m)
- **Standup Meeting** (estimated ~15m, actual)
- **QUIZ TIME!** (estimated ~20m, actual)
- **Homework Review** (estimated ~20m, actual)
- **Three Little Things** (estimated ~30m, actual)
### Control Flow: REDUX
- [ ] How do I alter the Control Flow of a program?
  - How do I _draw_ a decision tree? How do I _write_ one?
  - What if I have more than one decision to make?
  - How do I draw that?
- [ ] What if I need to make the same decision again?
### What is a Function: REDUX
- [x] How do I write a Function? How do I use one?
- [x] What are the 3 things a Function always has?
- [ ] Do I always have to write my own Functions?
  - What functions are built-in to JavaScript? (https://developer.mozilla.org/en-US/docs/Web/JavaScript)
  - Is there any other way to define or invoke a Function?
  - What is an anonymous function? What is it used for?
    - examples with `setTimeout`, `setInterval`
- [ ] How does this relate to Control Flow and the Rule of Sequential Operation?
### Data Types: Lists and Dictionaries
- [ ] What are the 4 primitive types in JavaScript?
- [ ] What if I need to store more than one value?
  - What is a list? How do I make one?
  - What is a dictionary? How do I make one?
  - How do I get values out?
  - How do I put new values in?
  - What can I put in a list or dictionary?
- [ ] What are they good for?

### CODE REVIEW

**Comment by al-the-x** [comment]
# Blockers
- Y U NO LIKE ANSWERS @al-the-x!?!?!
- little dim on `mocha` and `chai`...
- FRUSTRATION!
- BEAST MODE?
- Time Management
- Reading the Assignment
- How U even homework, GitHub?
- Writing... Hard.

**Comment by kellymurray** [comment]
### Three Little Things
1. Mocha/Chai - how do I get it working? Once it's working, what do I do with it?
2. Accessing homework via GitHub 
3. Clarity on the assignment - where do I start? The checkmarks seem vague! Help!

File: 03--The-Todo-Towers/README.md [hunk 1]
```diff
@@ -0,0 +1,306 @@
...
+    * [ ] thirty something: `33.33` into `"thirty three & 33/100s"`
+    * [ ] middle aged: `55.55` into `"fifty five & 55/100s"`
+  * **NIGHTMARE MODE** -- more `toCheck`
+    * [ ] literally hundreds: `123.45` into `"one hundred twenty three & 45/100s"`
+    * [ ] honestly hard: `1234.56` into `"one thousand, two hundred thirty four & 56/100s"`
+```
+
+### The Right Mindset
+
+Continue working on your essay, now progressing in the Revision phase. You should start to produce a Final Draft starting with the best parts of your Roughs. Edit and expand them with a series of revisions, which you save as commits in Github. You should keep your old Roughs and Outline around.```
```
Comment (al-the-x) [inline 03--The-Todo-Towers/README.md]: Little whoopsie... No backticks on the end of the line there, Mr Copy-Pasta.

File: 03--The-Todo-Towers/README.md [hunk 2]
```diff
@@ -0,0 +1,306 @@
...
+
+Using the MDN and your own research, describe [the control flow constructs](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Guide/Control_flow_and_error_handling): `{ }` as a block operator, `if` and `else` Then read and cover [the looping and iteration keywords](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Guide/Loops_and_iteration): `for`, `while`, and `do/while`.
+
+#### BEAST MODE
+
+If you're looking for more of a challenge, read up on `switch` / `case` and the other `for`-type loops. Document them, as well, providing examples.
+
+#### NIGHTMARE MODE
+
+The error handling constructs -- `try` / `catch` / `finally` and `throw` -- can be a bit puzzling. Review them if you're still not satisfied with your understanding of Control Flow. Be prepared for some wackiness.```
```
Comment (al-the-x) [inline 03--The-Todo-Towers/README.md]: Found some more copy-pasta, ya jerk.

**Comment by al-the-x** [comment]
[What is a Mulligan?](https://en.wikipedia.org/wiki/Mulligan_%28games%29)

**Pull request merged**